gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/pdeps/ProductDependencies.java

```diff
@@ -55,7 +58,29 @@ public static TaskProvider<ResolveProductDependenciesTask> registerProductDepend
                     params.getPathToExtract().set(RecommendedProductDependenciesPlugin.RESOURCE_PATH);
                 });
 
-        Provider<ArtifactView> discoveredDependencies = getDiscoveredDependencies(project, ext);
+        NamedDomainObjectProvider<Configuration> productDependencyDependencyScope = project.getConfigurations()
+                .register(PRODUCT_DEPENDENCY_DISCOVERY_CONFIGURATION_NAME, conf -> {
```

Comment:
When we only have to support a high enough version of Gradle, would this be `.dependencyScope`?